x-pack/plugins/alerting/server/task_runner/create_execution_handler.ts

```diff
@@ -192,6 +201,18 @@ export function createExecutionHandler<
             { type: 'action', id: action.id, type_id: action.actionTypeId, ...namespace },
           ],
         },
+        rule: {
+          id: alertType.id,
+          license: alertType.minimumLicenseRequired,
+          category: alertType.name,
+          ruleset: alertType.producer,
+          uuid: alertId,
+          reference: getDocsForRuleTypeByProducer(alertType.id, alertType.producer),
+          namespace: spaceId !== 'default' ? spaceId : undefined,
```

Comment:
I think `namespace` is already defined in line [179](https://github.com/elastic/kibana/pull/101132/files#diff-b54feaebf48fb1b7753cf65660e35ae72a49726c03db4f7133095854356d670cR179). Could that be reused here or is it different?

Reply:
Yeah, you are right - it is the same. Thank you!